pkg/config/log.go

```diff
@@ -52,29 +61,30 @@ func SetupLogger(logLevel, logFile, uri string, rfc, tls bool, pem string, logTo
 		var syslogTemplate string
 		if uri != "" {
 			syslogTemplate = fmt.Sprintf(
-				`<custom name="syslog" formatid="syslog" data-uri="%s" data-tls="%v" />`,
+				`<custom name="syslog" formatid="%s" data-uri="%s" data-tls="%v" />`,
+				formatID,
 				uri,
 				tls,
 			)
 		} else {
-			syslogTemplate = `<custom name="syslog" formatid="syslog" />`
+			syslogTemplate = fmt.Sprintf(`<custom name="syslog" formatid="%s" />`, formatID)
 		}
 		configTemplate += syslogTemplate
 	}
+
 	configTemplate += `</outputs>
-    <formats>
-        <format id="common" format="%%Date(%s) | %%LEVEL | (%%File:%%Line in %%FuncShort) | %%Msg%%n"/>`
+	<formats>
+		<format id="std:json" format="{&quot;time&quot;:&quot;%%Date(%s)&quot;,&quot;level&quot;:&quot;%%LEVEL&quot;,&quot;file&quot;:&quot;%%File&quot;,&quot;line&quot;:&quot;%%Line&quot;,&quot;func&quot;:&quot;%%FuncShort&quot;,&quot;msg&quot;:&quot;%%Msg&quot;}%%n"/>
+		<format id="common" format="%%Date(%s) | %%LEVEL | (%%File:%%Line in %%FuncShort) | %%Msg%%n"/>`
+
 	if syslog {
-		if rfc {
-			configTemplate += `<format id="syslog" format="%%CustomSyslogHeader(20,true) %%LEVEL | (%%RelFile:%%Line) | %%Msg%%n" />`
-		} else {
-			configTemplate += `<format id="syslog" format="%%CustomSyslogHeader(20,false) %%LEVEL | (%%RelFile:%%Line) | %%Msg%%n" />`
-		}
+		configTemplate += fmt.Sprintf(`<format id="std:json" format="{&quot;header&quot;:&quot;%%CustomSyslogHeader(20,%t)&quot;,&quot;level&quot;:&quot;%%LEVEL&quot;,&quot;relfile&quot;:&quot;%%RelFile&quot;,&quot;line&quot;:&quot;%%Line&quot;,&quot;msg&quot;:&quot;%%Msg&quot;}%%n"/>
```

Comment:
That would define two format with the id `std:json`. Could we name the syslog one something like `syslogjson` ?
So we would have 3 format: `common`, `json` and `syslogjson` and we could remove the `if syslog`.

What do you think ?